packages/tradershub/tsconfig.json

```diff
@@ -4,7 +4,9 @@
         "outDir": "./dist",
         "baseUrl": "./",
         "paths": {
-            "@deriv/*": ["../*/src"]
+            "@deriv/*": ["../*/src"],
```

Comment:
will it not conflict with other libraries like `@deriv/ui`?

Reply:
no because its deriv-com/ui

Reply:
i think it will not because `deriv/ui` is a package right
Which is not installed? 🤔

Reply:
we're not using anything related to `@deriv/*` like `@deriv/utils` or anything I guess. Right?

Reply:
Nopes.